sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -260,6 +260,24 @@ export function useFormatters() {
     return DEFAULT_EMPTY_VALUE;
   };
 
+  /**
+   * Converts the saved gender to the capitalized display version.
+   * @param gender value to be converted to be displayed.
+   * @returns formatted gender string.
+   */
+  const genderDisplayFormat = (gender: string): string | undefined => {
```

Comment:
Since the `switch` has a default, `undefined` will never be returned, right?

Reply:
ah, correct